physionet-django/user/models.py

```diff
@@ -749,13 +749,18 @@ class CredentialApplication(models.Model):
         (2, 'Yes')
     )
 
-    REJECT_ACCEPT_WITHDRAW = (
-        ('', '-----------'),
-        (1, 'Reject'),
-        (2, 'Accept'),
-        (3, 'Withdrawn'),
-        (4, 'Revoked')
-    )
+    class Status(models.IntegerChoices):
```

Comment:
Originally it was because the code before this was using integer, so my idea was if we use integerchoices we wont need to worry about the existing contents(as it was already integers and same values like right now)

In summary, mostly so that we dont make changes to the existing data.


But i learned about the migrate_forward,backward stuff few days ago, so if you prefer the textchoices, i can refactor the code.

Reply:
Thanks Amit, that's not a bad reason! I think it would be an improvement to switch to `TextChoices` if you think it wouldn't be too much extra effort? If we are going to make the change, now would be a good time. 

I notice `application.status` is referencing the numerical values in a few additional places (e.g. https://github.com/MIT-LCP/physionet-build/blob/dev/physionet-django/user/management/commands/test_reject_pending_credentialing_applications.py and https://github.com/MIT-LCP/physionet-build/blob/dev/physionet-django/user/templates/user/application_display_table.html so these would also need to be updated).

Reply:
@bemoody you mentioned that you thought keeping as integer would be easier/better? if so, please could you post your suggestion here before @amitupreti moves everything?

Reply:
@bemoody I hope you don’t mind me nudging you in case you missed @tompollard’s question.

Reply:
decision is to leave as integer type